tests/browserstack/specs/valuesMainTest.js

```diff
@@ -0,0 +1,210 @@
+  await simpleTextInput(searchBarId, searchValue); // Input text into search bar
+  await selectClick(searchButtonSelector); // Clicks search
+}
+
+async function readMore (selector) {
+  const selected = await $(selector);
+  const more = await selected.$('=More');
+  if (await more.isClickable()) {
+    await more.click();
+  }
+  await browser.pause(PAUSE_DURATION_MICROSECONDS);
+  const showLess = await selected.$('=Show Less');
+  if (await showLess.isClickable()) {
+    await browser.pause(PAUSE_DURATION_MICROSECONDS);
+    await showLess.click();
+  }
+  await browser.pause(PAUSE_DURATION_MICROSECONDS);
+}
+
+describe('Basic cross-platform We Vote test',  () => {
+  it('should load the app so we can run letious tests', async () => {
+    const { isCordovaFromAppStore, isMobileScreenSize } = driver.config.capabilities;
+    const isDesktopScreenSize = !isMobileScreenSize;
+
+    if (isCordovaFromAppStore) {
+    // ///////////////////////////////
+    // For the apps downloadable from either the Apple App Store or Android Play Store,
+    // click through the onboarding screens
+      await browser.pause(PAUSE_DURATION_MICROSECONDS * 2);
+      const contexts = await driver.getContexts();
+      const context = contexts.includes(ANDROID_CONTEXT) ? ANDROID_CONTEXT : IOS_CONTEXT;
+      await driver.switchContext(context);
+      await browser.pause(PAUSE_DURATION_MICROSECONDS);
+      const firstNextButton = await $('div[data-index="0"] .intro-story__btn--bottom');
+      await firstNextButton.click();
+      await browser.pause(PAUSE_DURATION_MICROSECONDS);
+      const secondNextButton = await $('div[data-index="1"] .intro-story__btn--bottom');
+      await secondNextButton.click();
+      await browser.pause(PAUSE_DURATION_MICROSECONDS);
+      const thirdNextButton = await $('div[data-index="2"] .intro-story__btn--bottom');
+      await thirdNextButton.click();
+      await browser.pause(PAUSE_DURATION_MICROSECONDS);
+    } else {
+    // ///////////////////////////////
+    // For the website version, open our quality testing site
+      await browser.url('https://quality.wevote.us/values');
+    }
+
+    await browser.pause(PAUSE_DURATION_MICROSECONDS);
+
+    const dropDownButton = 'button.dropdown-toggle.issues-follow-btn__dropdown';
+    const followButton = 'button.issues-follow-btn__main.issues-follow-btn--blue';
+    const unfollow = 'button.dropdown-item.issues-follow-btn__menu-item:first-child';
+    const ignore = 'button.dropdown-item.issues-follow-btn__menu-item:nth-of-type(2)';
+    const unignore = 'span.d-print-none button';
+    const link = 'a.u-no-underline';
+    const mainContentColumn = 'div.col-sm-12.col-md-8';
+    const mainContainer = 'div.opinions-followed__container';
+    const childCard = 'div.card-child.card-child--not-followed';
+    const clearSearchIconSelector = 'img[src="/img/glyphicons-halflings-88-remove-circle.svg"]';
+    const sqlInjectionTest = '\' or 1=1 -- -';
```

Comment:
Hi @4D617278 I really like that you are setting variables at the start of the testing script. Since we often rename styles, it will be more maintainable if you use id's. Feel free to add ids to the code as needed. For example instead of `const mainContentColumn = 'div.col-sm-12.col-md-8';` you can add `id='valuesMainContentColumn'` to that div and then reference that.